src/databricks/labs/ucx/source_code/linters/dbfs.py

```diff
@@ -21,30 +25,36 @@ def __init__(self):
 
     def visit_call(self, node: Call):
         for arg in node.args:
-            if isinstance(arg, Const) and isinstance(arg.value, str):
-                value = arg.value
-                if any(value.startswith(prefix) for prefix in self._fs_prefixes):
-                    deprecation = Deprecation.from_node(
-                        code='dbfs-usage', message=f"Deprecated file system path in call to: {value}", node=arg
-                    )
-                    self._advices.append(deprecation.replace(end_col=arg.col_offset + len(value)))
-                    # Record the location of the reported constant, so we do not double report
-                    self._reported_locations.add((arg.lineno, arg.col_offset))
+            self._visit_arg(arg)
+
+    def _visit_arg(self, arg: NodeNG):
+        try:
+            for inferred in arg.inferred():
+                if isinstance(inferred, Const) and isinstance(inferred.value, str):
+                    self._check_str_constant(arg, inferred)
+        except InferenceError:
+            logger.debug(f"Could not infer value of {str(arg)}")
```

Comment:
In format strings I don't think you need to wrap arguments in `str()` because that will happen anyway.

Also, might it be worth adding in `exc_info=True` here?

Reply:
done